components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -240,10 +241,11 @@ export function ConversationContextProvider(
     )
     listenerIds.push(id)
 
-    id = callbackRouter.onAPIResponseError.addListener((error: mojom.APIError) =>
-      setPartialContext({
-        currentError: error
-      })
+    id = callbackRouter.onAPIResponseError.addListener(
```

Comment:
please only format lines changed in a PR with non-formatting changes so it's easier to review and we don't make other pending PRs harder to rebase. e.g. I'm changing a lot of this file in another PR and if we're unneccessarily changing formatting it will waste time.

Reply:
What formatter do you use for *.tsx; I think VS Code might have munged up some of the files unintentionally when I attempted to format my own code. Apologies.

Reply:
Yes it does that. No good formatter at the moment, we're working on it and have something in-progress.